### ISSUE TITLE
Dragging and dropping work on firefox but not on chrome

### ISSUE THREAD

**Issue by MihaitaN** [heyscrumpy/tiptap#31]
`<template>
  <im-editor :extensions="extensions" @drop.native="aaa($event)">
    <div slot="content" slot-scope="props">
      <img src="https://media.giphy.com/media/ctNJp7PTRuRNTREyTj/giphy.gif">
      <p>Test Text</p>
    </div>
  </im-editor>`

**Comment by philippkuehn** [comment]
Can you provide a minimal test repo or an example at codesandbox.io?

**Comment by MihaitaN** [comment]
I can, hang on.

**Comment by philippkuehn** [comment]
Hey I can‘t find any drag and drop code in your example. 👀

**Comment by MihaitaN** [comment]
What do you mean? I just copied your example over from https://github.com/heyscrumpy/tiptap/blob/master/examples/Components/Routes/Images/index.vue .
It doesn't have any specific drag & drop logic, but I supposed editor component already handles this.

Try opening the codesandbox on chrome and firefox and dragging and dropping an image from your machine onto the editor component (between Test & Text works best :) ).

On firefox the image gets loaded into the editor component, but chrome just opens a new tab with the image you dropped (it navigates to the file address on your machine).

**Comment by philippkuehn** [comment]
Ah sorry, my fault 🙃

Hmm this is strange. Drop an image works fine on the official [images example](https://tiptap.scrumpy.io/images) but not for your example on codesandbox. I'll check it out.

**Comment by childonline** [comment]
Hey, it seems I stumbled over this exact same issue.

I'm also using chrome build 69.0.3497.100 and experiencing the same behavior.

The official image example does indeed work with chrome, but not when on localhost. Maybe it's something related to the build config of the examples page? I tried having a look, but couldn't find anything immediately obvious :disappointed: .

**Comment by childonline** [comment]
I just noticed this issue also present on prosemirror, you can replicate the drag & drop behavior over at http://prosemirror.net/examples/dino/ . :upside_down_face: So this is tehnically a ProseMirror issue not a TipTap issue.

But what really baffles me is why it works on chrome on the official images example.

**Comment by philippkuehn** [comment]
@childonline Yeah super strange. Didn't find anything until yet. The demo page setup is indeed different because I don't use the tiptap packages from npm. Instead I added some alias with webpack to grab these packages directly from the src folders of the repository. But I don't think that's what makes the difference.

**Comment by qyloxe** [comment]
I would bet that it has something to do with headers, origin, feature detection, accept or anything else linked to browser-server context. Chrome has lately a "dark" history for implementation of those things and breaking many previously working use cases. Of course, it is done in the name of "security" so nobody should complain.

**Comment by philippkuehn** [comment]
This is strange. In [this sandbox](https://codesandbox.io/s/13ml3zw8n4) I import a custom ImageNode extension (it's a copy from the image extension from tiptap-extensions) and it works fine. So it seems that something goes wrong in the build process? 🤷‍♂️

**Comment by childonline** [comment]
Yup, I can confirm, loading an identical "custom" ImageNode works on localhost aswell.

**Comment by childonline** [comment]
thx, mate!
